inc/composer/class-command.php

```diff
@@ -232,9 +232,27 @@ protected function shell( InputInterface $input, OutputInterface $output ) {
 			'cd %s; VOLUME=%s COMPOSE_PROJECT_NAME=%s docker-compose exec php /bin/bash',
 			'vendor/altis/local-server/docker',
 			getcwd(),
-			basename( getcwd() )
+			$this->get_project_subdomain()
 		), $return_val );
 
 		return $return_val;
 	}
+
+	/**
+	 * Get the name of the project for the local subdomain
+	 *
+	 * @return string
+	 */
+	protected function get_project_subdomain() : string {
+
+		$composer_json = json_decode( file_get_contents( getcwd() . '/composer.json' ), true );
+
+		if ( isset( $composer_json['extra']['altis']['modules']['local-server']['name'] ) ) {
```

Comment:
I thought you had to check each stage of the array tree when using `isset()` so I've used null coalescing in most cases like this. If there're no warnings then TIL!

Reply:
Nope! https://3v4l.org/9e8R5